pkg/Cpanel/Security/Advisor/Assessors/Mysql.pm

```diff
@@ -79,14 +78,17 @@ sub _check_for_db_test {
     if ( !$exists ) {
         $self->add_good_advice(
             'key'  => 'Mysql_test_database_does_not_exist',
-            'text' => $self->_lh->maketext("MySQL test database doesn't exist.")
+            'text' => $self->_lh->maketext("[asis,MySQL] test database doesn’t exist.")
         );
     }
     else {
         $self->add_bad_advice(
             'key'        => 'Mysql_test_database_exists',
-            'text'       => $self->_lh->maketext("MySQL test database exists."),
-            'suggestion' => $self->_lh->maketext(q{MySQL test database is used by numerous attacks and should be removed by running “mysql -e 'drop database test'”.}),
+            'text'       => $self->_lh->maketext("[asis,MySQL] test database exists."),
+            'suggestion' => $self->_lh->maketext(
+                '[asis,MySQL] test database is used by numerous attacks and should be removed by running “[_1]”.',
```

Comment:
Numerous attacks exploit the [asis,MySQL] test database. To remove it, run “[_1]”.